core/tx_pool.go

```diff
@@ -157,7 +157,7 @@ var DefaultTxPoolConfig = TxPoolConfig{
 	Journal:   "transactions.rlp",
 	Rejournal: time.Hour,
 
-	PriceLimit: gasprice.Default.Uint64(),
+	PriceLimit: 0,
```

Comment:
Fixed value defaults and sanitizing too early (below) made it impossible to tell later on where the value came from. Now we defer until later so that the default can be dynamic and change with the fork block.